Tools/Scripts/libraries/webkitcorepy/webkitcorepy/autoinstall.py

```diff
@@ -292,7 +292,7 @@ def install(self):
             AutoInstall.log('Downloading {}...'.format(archive))
             archive.download()
 
-            temp_location = os.path.join(tempfile.gettempdir(), self.name)
+            temp_location = os.path.join(tempfile.gettempdir(), '{}-{}'.format(self.name, os.getpid()))
```

Comment:
This is an improvement, but it's pretty unconventional. Can we use `tempfile.TemporaryDirectory`?

Reply:
One of `webkitcorepy`'s important functions is to provide Python 3 idioms to Python 2, and (per https://docs.python.org/2.7/library/tempfile.html), we don't have `TemporaryDirectory` in Python 2.

Reply:
Good point, `tempfile.mkdtemp` then.